codegen/type_converter.go

```diff
@@ -71,14 +71,26 @@ type TypeConverter struct {
 	Helper        PackageNameResolver
 	uninitialized map[string]*fieldStruct
 	fieldCounter  int
+
+	requestTypeHelper ConvertOptions
+}
+
+// ConvertOptions is the helper struct for method generation
+type ConvertOptions struct {
+	FromSuffix       string
+	FromInputType    string
+	FromOutputType   string
+	ToType           string
+	OutputMethodName string
 }
 
 // NewTypeConverter returns *TypeConverter
-func NewTypeConverter(h PackageNameResolver) *TypeConverter {
+func NewTypeConverter(h PackageNameResolver, requestType ConvertOptions) *TypeConverter {
```

Comment:
paramName and typename dont read cleanly.

Reply:
what do you suggest